packages/core/src/recon.ts

```diff
@@ -57,7 +57,7 @@ export interface ReconEventFeedResponse {
  */
 export interface IReconApi extends Observable<ReconEventFeedResponse> {
   init(initialCursor?: string, initialInterests?: Array<StreamID>): Promise<void>
-  registerInterest(model: StreamID): Promise<void>
+  registerInterest(model: StreamID, controller?: string): Promise<void>
```

Comment:
this should probably be kept separate

Reply:
I don't really see why tbh?  It's a very small change

Reply:
yeah its an important one tho and if we wanted to roll back node metrics we wouldn't want to lose dids as a part of interests...?

Reply:
not sure what you mean by "lose dids as part of interests".  This is just a small change to the client to expose a feature of C1 that exists either way.  Right now the NodeMetrics are the only consumer of this API, nothing else is using it so it'd be fine roll back if the NodeMetrics rolls back.